hack/generate-manifest-flow-visibility.sh

```diff
@@ -20,11 +20,15 @@ function echoerr {
     >&2 echo "$@"
 }
 
-_usage="Usage: $0 [--mode (dev|release)] [--keep] [--help|-h]
+_usage="Usage: $0 [--mode (dev|release|e2e)] [--keep] [--help|-h]
 Generate a YAML manifest for the Clickhouse-Grafana Flow-visibility Solution, using Kustomize, and
 print it to stdout.
-        --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
-        --keep                Debug flag which will preserve the generated kustomization.yml
+        --mode (dev|release|e2e)  Choose the configuration variant that you need (default is 'dev')
+                                  e2e mode generates YAML manifest for e2e test, which includes
+                                  clickhouse operator and server with default credentials,
+                                  but not Grafana-related functionality
```

Comment:
It will be good to mention that the ClickHouse monitor is not included too.

Reply:
monitor is actually included in the test